pkg/actors/internal/api_level.go

```diff
@@ -27,10 +27,14 @@ const ActorAPILevel = 20
 type apiLevelFeature uint32
 
 const (
-	// Enables serializing reminders as protobuf rather than JSON in the pkg/actors/reminders package
+	// APILevelFeatureRemindersProtobuf Enables serializing reminders as protobuf rather than JSON in the pkg/actors/reminders package
 	// When serialized as protobuf, reminders have the "\0pb" prefix
 	// Note this only control serializations; when un-serializing, legacy JSON is always supported as fallback
 	APILevelFeatureRemindersProtobuf apiLevelFeature = 20
+
+	// NoVirtualNodesInPlacementTables - The placement table doesn't include the virtual nodes
+	// Instead, they're calculated on the client (daprd) side
+	NoVirtualNodesInPlacementTables apiLevelFeature = 20
```

Comment:
Why the same number as above?

Reply:
This indicates the API level. For Dapr 1.12, it's 10. In 1.13 it will be 20. #6838

Reply:
I am new to this one. Thanks.